android/src/test/java/com/marigold/rnsdk/RNMessageStreamModuleTest.kt

```diff
@@ -90,7 +106,11 @@ class RNMessageStreamModuleTest {
         doReturn(jsonObject).whenever(message).toJSON()
         doThrow(jsonException).whenever(jsonConverter).convertJsonToMap(jsonObject)
 
-        val shouldPresent = rnMessageStreamModuleSpy.shouldPresentInAppNotification(message)
+        rnMessageStreamModuleSpy.useDefaultInAppNotification(false)
+        val shouldPresent = runBlocking {
```

Comment:
Do we need a runBlocking here? `shouldPresentInAppNotification` isn't a suspend method.